### PR TITLE
Add content-info Quarkus CLI command

### PR DESCRIPTION
**Pull request by dimas-b**
* This command can help to analyse how data is stored in older repositories that was produced by older Nessie versions.

In particular, this command can detect the presence of content objects using the obsolete "global state" storage model.

* Also factor out common test code into BaseContentTest

### CODE REVIEW

File: servers/quarkus-cli/src/main/java/org/projectnessie/quarkus/cli/ContentInfo.java [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+              batch.add(keyListEntry.getKey());
+
+              if (batch.size() >= batchSize) {
+                check(head, batch, generator);
+                batch.clear();
+              }
+            });
+      }
+
+      check(head, batch, generator); // check remaining keys
```
Comment (snazy) [inline servers/quarkus-cli/src/main/java/org/projectnessie/quarkus/cli/ContentInfo.java:169]: Should be guarded with a `if (!batch.isEmpty())`

Comment (dimas-b) [reply servers/quarkus-cli/src/main/java/org/projectnessie/quarkus/cli/ContentInfo.java]: good catch - fixed

**Review by snazy — DISMISSED** [review]

**Comment by dimas-b** [comment]
Local build - similar:
```
$ ./gradlew :nessie-quarkus-cli:check
Configuration on demand is an incubating feature.

> Task :nessie-quarkus-cli:test
Generated Jacoco reports in /home/dmitri/prj/nessie/servers/quarkus-cli/build/jacoco-report
Generated Jacoco reports in /home/dmitri/prj/nessie/servers/quarkus-cli/build/jacoco-report

> Task :nessie-quarkus-cli:intTest
[168.239s][warning][gc,alloc] build-53: Retried waiting for GCLocker too often allocating 9968 words

Exception: java.lang.OutOfMemoryError thrown from the UncaughtExceptionHandler in thread "build-211"
[195.293s][warning][gc,alloc] build-99: Retried waiting for GCLocker too often allocating 3115 words
[195.293s][warning][gc,alloc] build-166: Retried waiting for GCLocker too often allocating 3625 words
[197.204s][warning][gc,alloc] build-81: Retried waiting for GCLocker too often allocating 6039 words

Exception: java.lang.OutOfMemoryError thrown from the UncaughtExceptionHandler in thread "build-225"

Exception: java.lang.OutOfMemoryError thrown from the UncaughtExceptionHandler in thread "build-16"

Exception: java.lang.OutOfMemoryError thrown from the UncaughtExceptionHandler in thread "build-222"
[231.114s][warning][gc,alloc] build-203: Retried waiting for GCLocker too often allocating 9058 words
[232.218s][warning][gc,alloc] build-200: Retried waiting for GCLocker too often allocating 8304 words

Exception: java.lang.OutOfMemoryError thrown from the UncaughtExceptionHandler in thread "build-243"
<=<============-> 93% EXECUTING [5m 6s]
> :nessie-quarkus-cli:intTest > 28 tests completed
> :nessie-quarkus-cli:intTest > Executing test org.projectnessie.quarkus.cli.ITExportImport
```

**Comment by dimas-b** [comment]
Opened #5530 to address the out-of-memory problem

**Review by snazy — APPROVED** [review]

**Pull request merged**